tinygrad/renderer/cstyle.py

```diff
@@ -311,12 +308,12 @@ def __reduce__(self): return self.__class__, (self.arch,)
   float4 = "make_float4"
   code_for_workitem = {"g": lambda x: f"blockIdx.{chr(120+int(x))}", "l": lambda x: f"threadIdx.{chr(120+int(x))}",
                        "i": lambda x: f"(blockIdx.{chr(120+int(x))}*blockDim.{chr(120+int(x))}+threadIdx.{chr(120+int(x))})"}
-  code_for_op = {**CStyleLanguage.code_for_op,
-    UnaryOps.RECIP: lambda x,dtype: f"hrcp({x})" if dtype in (dtypes.half, dtypes.bfloat16) else f"(1/{x})",
-    UnaryOps.SQRT: lambda x,dtype: f"hsqrt({x})" if dtype in (dtypes.half, dtypes.bfloat16) else f"sqrt({x})",
+  code_for_op = { **CStyleLanguage.code_for_op,
```

Comment:
any reason to reorder these (and the one below)?

Reply:
just for readability, sin, they are visually aligned and stay coherently ordered

Reply:
> just for readability, they are visually aligned and stay coherently ordered

same as for cstyle, no functional reason either, just thought it ended up cleaner

Reply:
oh it's sorted by name length... fine i guess